library/src/lib/utils/dynamic-component/dynamic-component.service.ts

```diff
@@ -0,0 +1,77 @@
+import {
+    ComponentFactoryResolver,
+    Injectable,
+    ApplicationRef,
+    Injector,
+    EmbeddedViewRef,
+    ComponentRef,
+    Type, TemplateRef
+} from '@angular/core';
+import { DynamicComponentInjector } from './dynamic-component-injector';
+import { DynamicComponentConfig } from './dynamic-component-config';
+
+/**
+ * Service used to dynamically generate a modal.
```

Comment:
Service used to dynamically generate components like modals/alerts/notifications